README.md

```diff
@@ -1,19 +1,19 @@
-# magicbell-python-sdk
+# MagicBell-Python
 
-![magicbell logo purple](./assets/MB_logo_Purple_2800x660.png)
+<img src="./assets/MB_logo_Purple_2800x660.png" width="50%">
 
-An unofficial Python SDK for [MagicBell](https://magicbell.com).
+This SDK provides convenient access to the [MagicBell REST API](https://magicbell.com/docs/rest-api/overview) from applications written in Python. It includes helpers for creating notifications, managing users, managing projects, and GraphQL.
 
-[Install](#installation--usage) | [Getting Started](#getting-started) | [Examples](./examples) | [License](./LICENSE) | [Code of Conduct](./CODE_OF_CONDUCT.md) | [Contributing](./CONTRIBUTING.md)
+This is an unofficial Python SDK for [MagicBell](https://magicbell.com) open sourced with ❤️ by <a href="https://noteable.io">Noteable</a>, a collaborative notebook platform that enables teams to use and visualize data, together.
+
+[Install](#installation) | [Getting Started](#getting-started) | [Examples](./examples) | [License](./LICENSE) | [Code of Conduct](./CODE_OF_CONDUCT.md) | [Contributing](./CONTRIBUTING.md)
 
-- API Version: 1.0
```

Comment:
I would put these in the release section instead of putting them in the description. The release section makes it more structured for GitHub's algo's to pick up new releases etc.